ocsmesh/engines/jigsaw.py

```diff
@@ -37,6 +38,7 @@ def __init__(
 
         self._hfun_marche = hfun_marche
         self._remesh_tiny = remesh_tiny_elements
+        self._bnd_representation = bnd_representation
```

Comment:
The same this as above applies here. We don't need to store this separately as we don't want to pass it on